### PR TITLE
Prep for travis deploy

### PR DESCRIPTION
**Pull request by austinpapp**
This will be used to get travis ready for building the artifact we will upload to s3. this will be used during the new instance provisioning and deployment methods. 

/cc @igaponov

### CODE REVIEW

**Comment by austinpapp** [comment]
@igaponov this should not cause any issues while we still use teamcity for build and deployment. this should prepare us to move to the new method _when we are ready._

File: backend/scripts/before-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    --exclude='src/Civix/FrontBundle/Test*' \
+    --include='src/***' \
+    --exclude='web/app_test.php' \
+    --include='web/***' \
+    --include='vendor/***' \
+    --include='composer.*' \
+    --exclude='deployment/' \
+    --exclude='*' \
+    ./ $BUILD_DIR
+
```
Comment (igaponov) [inline backend/scripts/before-deploy.sh]:
```
rsync -qavz \
    --include='app/***' \
    --include='bin/***' \
    --include='var/bootstrap.php.cache' \
    --exclude='src/Civix/*Bundle/Test*' \
    --include='src/***' \
    --exclude='web/app_test.php' \
    --include='web/***' \
    --include='vendor/***' \
    --include='composer.*' \
    --exclude='deployment/' \
    --exclude='*' \
    ./ $BUILD_DIR
```

Comment (austinpapp) [inline backend/scripts/before-deploy.sh]: @igaponov did you test this ?

Comment (austinpapp) [inline backend/scripts/before-deploy.sh]: i ask because why include the `app/cache` and `app/log` dirs and the `phpunit.xml.dist` ?

**Comment by igaponov** [comment]
@austin there are no cache, log dirs and phpunit file in app dir in new
version of symfony. I took this config from build.xml and it works now on
dev.

**Comment by austinpapp** [comment]
@igaponov cleaned up rsync

**Pull request merged**